spec/plaintime.html

```diff
@@ -885,7 +885,8 @@ <h1>ToTemporalTimeRecord ( _temporalTimeLike_ )</h1>
           1. Let _property_ be the Property value of the current row.
           1. Let _value_ be ? Get(_temporalTimeLike_, _property_).
           1. If _value_ is *undefined*, then
-            1. Throw a *TypeError* exception.
+            1. If _someRequired_ is not *false*, then
```

Comment:
If I'm reading correctly, this will behave differently from the code. This would throw the exception if _any_ properties were missing, rather than if _all_ properties were missing.

Reply:
oops, correct; will fix

Reply:
Fixed in update